src/SprykerEco/Zed/Computop/Business/ComputopFacadeInterface.php

```diff
@@ -112,6 +111,9 @@ public function refundCommandHandle(array $orderItems, OrderTransfer $orderTrans
 
     /**
      * Specification:
+     * - Requires QuoteTransfer::payment to be set.
```

Comment:
1. Please use backticks (`) to surround transfers
2. Please use `QuoteTransfer.payment` pattern
3. Single line with `QuoteTransfer.payment.computopSofort.sofortInitResponse.header` is a correct way for this case
same for similar cases across PR

Reply:
Fixed